packages/types/src/interfaces/babe/definitions.ts

```diff
@@ -5,12 +5,12 @@
 export default {
   types: {
     BabeWeight: 'u64',
-    SlotNumber: 'u64',
     RawBabePreDigest: {
       slotNumber: 'SlotNumber',
       authorityIndex: 'u32', // AuthorityIndex (also in aura, not same size there)
       vrfOutput: 'H256', // should be '[u8; 32]' (generator support lacking here)
       vrfProof: 'H256' // should be '[u8; 32]'
-    }
+    },
```

Comment:
From here going forward, we are only re-arranging things to be alphabetical - since we can now assign in any order, make sure that this is actually the case (testing needs to be in-place, so if the ordering is an issue, this would create chaos...)